unit_tests/test_scylla_yaml_builders.py

```diff
@@ -543,48 +543,40 @@ def validate_scylla_yaml(self, expected_node_config, seed_node_ips):
 
 
 class IntegrationTests(unittest.TestCase):
-    get_scylla_ami_version_output = ObjectDict(**{
-        'Architecture': 'x86_64', 'CreationDate': '2022-10-13T13:17:17.000Z',
-        'ImageId': 'ami-0dfb316a2cc0ab399', 'ImageLocation': '797456418907/ScyllaDB Enterprise 2021.1.15',
-        'ImageType': 'machine', 'Public': True, 'OwnerId': '797456418907',
-        'PlatformDetails': 'Linux/UNIX', 'UsageOperation': 'RunInstances',
-        'State': 'available', 'BlockDeviceMappings': [
-            {'DeviceName': '/dev/sda1',
-             'Ebs': {'DeleteOnTermination': True,
-                     'SnapshotId': 'snap-0717a2bee0a38bc84',
-                     'VolumeSize': 30,
-                     'VolumeType': 'gp3',
-                     'Encrypted': False}},
-            {'DeviceName': '/dev/sdb',
-             'VirtualName': 'ephemeral0'},
-            {'DeviceName': '/dev/sdc',
-             'VirtualName': 'ephemeral1'},
-            {'DeviceName': '/dev/sdd',
-             'VirtualName': 'ephemeral2'},
-            {'DeviceName': '/dev/sde',
-             'VirtualName': 'ephemeral3'},
-            {'DeviceName': '/dev/sdf',
-             'VirtualName': 'ephemeral4'},
-            {'DeviceName': '/dev/sdg',
-             'VirtualName': 'ephemeral5'},
-            {'DeviceName': '/dev/sdh',
-             'VirtualName': 'ephemeral6'},
-            {'DeviceName': '/dev/sdi',
-             'VirtualName': 'ephemeral7'}],
-        'Description': 'ScyllaDB Enterprise 2021.1.15', 'EnaSupport': True, 'Hypervisor': 'xen',
-        'Name': 'ScyllaDB Enterprise 2021.1.15', 'RootDeviceName': '/dev/sda1', 'RootDeviceType': 'ebs',
-        'Tags': [
-            {'Key': 'ScyllaMachineImageVersion', 'Value': '2021.1.15-20221011.82741b636ba-1'},
-            {'Key': 'ScyllaPython3Version', 'Value': '2021.1.15-20221011.82741b636ba-1'},
-            {'Key': 'user_data_format_version', 'Value': '2'},
-            {'Key': 'ScyllaToolsVersion', 'Value': '2021.1.15-20221011.82741b636ba-1'},
-            {'Key': 'ScyllaJMXVersion', 'Value': '2021.1.15-20221011.82741b636ba-1'},
-            {'Key': 'branch', 'Value': 'branch-2021.1'},
-            {'Key': 'scylla-git-commit', 'Value': '3d8c23d0b20af12302608c6286ace0c3dc070828'},
-            {'Key': 'build-tag', 'Value': 'jenkins-enterprise-2021.1-promote-release-213'},
-            {'Key': 'ScyllaVersion', 'Value': '2021.1.15-20221011.82741b636ba-1'},
-            {'Key': 'build-id', 'Value': '213'}
-        ], 'VirtualizationType': 'hvm'})
+    get_scylla_ami_version_output = ObjectDict(Architecture='x86_64', CreationDate='2022-10-13T13:17:17.000Z', ImageId='ami-0dfb316a2cc0ab399', ImageLocation='797456418907/ScyllaDB Enterprise 2021.1.15', ImageType='machine', Public=True, OwnerId='797456418907', PlatformDetails='Linux/UNIX', UsageOperation='RunInstances', State='available', BlockDeviceMappings=[
```

Comment:
Can't agree with this change.  In this particular case removing of `**` is enough because ObjectDict is a subclass of `dict`

Reply:
I has nothing todo with what is `ObjectDict`, `**` takes a dict and expend it into argument of a call.

and this is a style related linting rule, the it's more readable to pass the parameter as is in this case, and not expanding them via `**`